msprime/mutations.py

```diff
@@ -308,8 +333,24 @@ def __init__(self, kappa, equilibrium_frequencies=None, root_distribution=None):
 
 class GTRMutationModel(MatrixMutationModel):
     """
-    The Generalised time-reversible mutation model (Tavaré et al. 1986).
-    .. todo: documentation
+    The Generalised Time-Reversible nucleotide mutation model, a general
+    parameterization of a time-reversible mutation process (Tavaré et al.
+    1986). It allows specification of per-nucleotide equilibrium frequencies
+    and equilibrium transition rates.
+
+    See the documentation for more detail on the precise
+    parameterization.
+
+    :param array-like[float] relative_rates: Controls the relative rates of
+        mutation between each pair of nucleotides, in order "A<->C", "A<->T",
+        "C<->G", "C<->T", and "G<->T".
```

Comment:
This part is missing a set of nucleotides:
```suggestion
    :param array-like[float] relative_rates: Controls the relative rates of
        mutation between each pair of nucleotides, in order "A<->C", "A<->G", "A<->T",
        "C<->G", "C<->T", and "G<->T".
```